content/kotlin/concepts/datetime/terms/atTime/atTime.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: 'Understanding the .atTime() Method in Kotlin'
+Description: 'Learn about the .atTime() method in Kotlin for combining date and time.'
+Subjects:
+  - 'Kotlin'
+  - 'Programming'
+Tags:
+  - 'Date'
+  - 'Time'
+  - 'Kotlin'
+CatalogContent:
+ - 'learn-kotlin'
+ - 'paths/computer-science'
+---
+
+# .atTime() Method in Kotlin
+
+The **`.atTime()`** method in Kotlin is used to combine a date and a time, resulting in a `LocalDateTime` object. This is particularly useful when you need to represent both date and time components together.
+
+## Syntax
+
+```kotlin
+fun LocalDate.atTime(hour: Int, minute: Int, second: Int): LocalDateTime
+fun LocalDate.atTime(time: LocalTime): LocalDateTime
+fun LocalDate.atTime(time: LocalTime, second: Int): LocalDateTime
+```
```

Comment:
```suggestion
```pseudo
fun LocalDate.atTime(hour: Int, minute: Int, second: Int): LocalDateTime
fun LocalDate.atTime(time: LocalTime): LocalDateTime
fun LocalDate.atTime(time: LocalTime, second: Int): LocalDateTime
```
```
Give a line space before and after a code block or syntax block. Remember `pseudo` is used for syntax blocks.